.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "svg.preview.background": "transparent"
+}
```

Comment:
Unrelated ;-)

Reply:
Don't really know how this was committed, but anyway I just started all over again :)